plugins/modules/meraki_ms_switchport.py

```diff
@@ -646,6 +652,7 @@ def main():
             payload["stickyMacAllowList"] = macs
             payload["stickyMacAllowListLimit"] = sticky_mac_limit
         payload = clear_vlan(meraki.params, payload)
+        payload = clear_polices(meraki.params, payload)
```

Comment:
Will this always execute the `clear_policies` function? My preference is it's contained in an if statement to make it more obvious what happens (or doesn't happen).

Reply:
No, due to I check the param `params["access_policy_type"]`, and that if statement is in the definition of function. 
```python
def clear_polices(params, payload):
    if params["access_policy_type"] is None:
        payload["accessPolicyType"] = "Open"
    return payload
```